pkg/security/probe/probe_ebpf.go

```diff
@@ -1111,6 +1122,12 @@ func (p *EBPFProbe) handleEvent(CPU int, data []byte) {
 	// resolve the container context
 	event.ContainerContext, _ = p.fieldHandlers.ResolveContainerContext(event)
 
+	// send related events
+	for _, relatedEvent := range relatedEvents {
+		p.DispatchEvent(relatedEvent, true)
+	}
```

Comment:
should we be concerned about having to process a lot of events here (like at startup?), to the point it could have other bad side effects (slows, drops etc)?

Reply:
Per the test I did, I don't expect to see a lot of events generated.